docs/websites.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Creating serverless PHP websites
+currentMenu: php
+introduction: Learn how to deal with assets and static files to deploy serverless PHP websites.
+---
+
+Websites usually contain 2 parts:
+
+- PHP code, which runs on AWS Lambda and is [served by API Gateway](/docs/runtimes/http.md)
+- static files, which can be hosted and [served by AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html)
+
+It is possible to use these services with custom domains to build websites, however we quickly face the following limitations:
+
+- API Gateway is *HTTPS only*
+- S3 is *HTTP only*
+
+A simple way to support both HTTP and HTTPS is to use a free CDN like [Cloudflare.com](https://www.cloudflare.com/).
```

Comment:
Wouldn't AWS CloudFront make a better suggestion for providing HTTPS support on a bucket?

Reply:
CloudFront would definitely work and be a good option, but it is a bit complex. Cloudflare has a very simple UI: you basically sign up, add the website and setup your DNS and that's it (and it's free).

I'm not sure about suggesting Cloudflare though, it might be more confusing than anything for those who don't know about it 🤔

Reply:
I've removed mentions of Cloudflare, indeed it is better to stay with AWS and use CloudFront.